packages/start/shared/ErrorBoundary.tsx

```diff
@@ -17,31 +18,37 @@ function ErrorMessage(props: { error: any }) {
   createEffect(() => console.error(props.error));
 
   return (
-    <div style="padding: 16px">
+    <div style={{ padding: '16px' }}>
+      <HttpStatusCode code={500} />
       <div
-        style={`
-          background-color: rgba(252, 165, 165);
-          color: rgb(153, 27, 27);
-          border-radius": 5px;
-          overflow: scroll;
-          padding: 16px;
-          margin-bottom: 8px;
-        `}
+        style={{
+          'background-color': 'rgba(252,165,165)',
+          color: 'rgb(153,27,27)',
+          'border-radius': '5px',
+          overflow: 'scroll',
+          padding: '16px',
+          'margin-bottom': '8px',
+        }}
       >
-        <p style="font-weight: bold" id="error-message">
+        <p style={{ 'font-weight': 'bold' }} id="error-message">
           {props.error.message}
         </p>
         <button
           id="reset-errors"
           onClick={resetErrorBoundaries}
-          style={`color: rgba(252, 165, 165);
-            background-color: rgb(153, 27, 27);
-            border-radius: 5px;
-            padding: 4px 8px`}
+          style={{
+            color: 'rgba(252, 165, 165)',
+            'background-color': 'rgb(153, 27, 27)',
+            'border-radius': '5px',
+            padding: '4px 8px',
+            'margin-top': '8px',
```

Comment:
With some space between error message and retry button looks prettier

Reply:
I actually changed all these styles to strings intentionally to reduce compiled size output.

Reply:
Changed back to string.
I just thought styles are compiled to string during the build time...